viewer/core/src/public/migration/Cognite3DViewer.test.ts

```diff
@@ -224,14 +224,16 @@ describe('Cognite3DViewer', () => {
 
     // Act
     viewer.setCameraControlsOptions({
-      onClickTargetChange: !originalCameraControlsOptions.onClickTargetChange,
+      changeCameraTargetOnClick: !originalCameraControlsOptions.changeCameraTargetOnClick,
       mouseWheelAction: 'zoomToTarget'
     });
 
     // Assert
     const newCameraControlsOptions = viewer.getCameraControlsOptions();
 
-    expect(newCameraControlsOptions.onClickTargetChange).not.toEqual(originalCameraControlsOptions.onClickTargetChange);
+    expect(newCameraControlsOptions.changeCameraTargetOnClick).not.toEqual(
+      originalCameraControlsOptions.changeCameraTargetOnClick
+    );
```

Comment:
This test doesn't look very useful, it basically just checks if a setter owrks. Not sure there's any good way to make it better though

Reply:
Could be a check for added eventListeners maybe, can look into that in other PR.